### PR TITLE
Log poller on avax hot fix

### PR DESCRIPTION
**Pull request by connorwstein**

### CODE REVIEW

File: core/chains/evm/logpoller/clients.go [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	var elgs []types.Log
+	for _, lg := range lgs {
+		lgs = append(lgs, lg)
+	}
+	return elgs, err
+}
+
+func (a *avaClient) BatchCallContext(ctx context.Context, b []rpc.BatchElem) error {
+	//TODO implement me
+	panic("implement me")
```
Comment (connorwstein) [inline core/chains/evm/logpoller/clients.go:67]: will be needed for the VRF, not needed to unblock

Comment (connorwstein) [reply core/chains/evm/logpoller/clients.go:67]: log poller doesn't do any sends

Comment (jmank88) [reply core/chains/evm/logpoller/clients.go:68]: Oh I see this is a new, separate client instance... :thinking:

File: core/chains/evm/logpoller/clients.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+	})
+	if err != nil {
+		return nil, err
+	}
+	// Cast to geth type
+	var elgs []types.Log
+	for _, lg := range lgs {
+		lgs = append(lgs, lg)
+	}
+	return elgs, err
```
Comment (MStreet3) [inline core/chains/evm/logpoller/clients.go:63]:
```suggestion
	return elgs, nil
```

**Comment by connorwstein** [comment]
Closing in favor of https://github.com/smartcontractkit/chainlink/pull/7673

**Pull request closed without merging**